platform/device/builtin/db.go

```diff
@@ -46,6 +47,24 @@ func NewDB(db *bolt.DB) (*DB, error) {
 	return datastore, nil
 }
 
+func (db *DB) GetBootstrapToken(ctx context.Context, udid string) ([]byte, error) {
+	opts := device.ListDevicesOption{}
+	
+	var udids []string
+	udids = append(udids, udid)
+
+	opts.FilterUDID = udids
+	
+	devices, err := db.List(ctx, opts)
```

Comment:
We should avoid `db.List` here. It enumerates every device and only filters after the fact. We should instead use `db.DeviceByUDID()` which then obviates the need for all the UDID stuff.